docs/ocis/guides/migrate-data-rclone.md

```diff
@@ -0,0 +1,103 @@
+
+This article explains by the example of Nextcloud how you would migrate your data from an running NC to Infinite Scale. A prerequisite is that you have Infinite Scale already set up and running on a different network address.
+
+{{< hint warning >}}
+Be prepared that migration can take some time. Also, check the size of your data. This example was around 1.5 GiB of data, that obviously went fast.
+
+And of course: Have a backup! Even if this method only reads from the source, you never know.
+{{< /hint >}}
+
+## Install rclone
+
+Check the [rclone website on how to install](https://rclone.org/install/) rclone.
+
+## Create Users
+
+First, decide on a user that you want to migrate. To create the user on Infinite Scale, log in as an admin user and create the desired user. Create and set groups accordingly.
+
+For the next step, you need the user credentials on both the source- and the destination cloud.
+
+## Configure rclone Remotes
+
+To be able to address the clouds in rclone, you need to configure so called _remotes_. It is nothing else than a shortcut for the combination of
+- which kind of cloud are you talking to
+- the URL
+- username
+- evtl. password
+
+You need to add a configuration for both the source cloud (Nextcloud) and the target (Infinite Scale). Both talk WebDAV, so [this manual page](https://rclone.org/webdav/) is accurate to follow.
+
+For both, use an URL in the form of `https://my.host.de/remote.php/webdav`.
+
+Once that is finished, the command `rclone config show` should give output similar to this:
+
+```bash
+[:~/] ± rclone config show
+
+[NCoC]
+type = webdav
+url = https://nc.this.de/remote.php/webdav
+vendor = nextcloud
+user = wilma
+pass = zfdsaiewrafdskfjdasfxdasffdafdsafas
+
+[ocis]
+type = webdav
+url = https://infinitescale.works/remote.php/webdav
+vendor = owncloud
+user = wilma
+pass = cdsfasrefdsadaGkxTXjksfpqQFI5nQawqs
+
+```
+
+Now, for example the directories on the Nextcloud root can be checked with `rclone lsd NCoC:/`.
+
+## Copy Data
+
+To migrate the data, rclone knows the very useful command `copy`. It transfers data from one remote to the other. So to transfer the entire cloud data from Nextcloud to Infinite Scale, this is the command:
+```
+rclone  copy NCoC:/ ocis:/ --no-check-certificate -P
+```
+The --no-check-certificate can and should be skipped if your clouds have proper certificates. The `-P` however, provides you with interesting statistics about the copy progress.
+Once you are finished, this might be the result:
+```
+[:~/] $ rclone copy NCoC:/ ocis:/ --no-check-certificate -P
+Transferred:        1.228 GiB / 1.228 GiB, 100%, 10.170 MiB/s, ETA 0s
+Transferred:          411 / 411, 100%
+Elapsed time:      2m19.3s
+```
+
+Note that while testing this, occasionally the Nextcloud was returning a `04 not found` for some files. While the reason for that was not completely clear, it does not matter, because the rclone command can be repeated. It is clever enough to only copy what has changed!
```

Comment:
```suggestion
Note that while testing this, occasionally the Nextcloud was returning a `404 not found` for some files. While the reason for that was not completely clear, it does not matter, because the rclone command can be repeated. It is clever enough to only copy what has changed!
```